src/main/app/forms/models/address.ts

```diff
@@ -62,8 +67,10 @@ export class Address implements CompletableTask {
   city?: string
 
   @ValidateIf(o => o.addressVisible, { groups: ['claimant', 'defendant', 'response'] })
-  @IsDefined({ message: ValidationErrors.POSTCODE_REQUIRED, groups: ['claimant', 'defendant', 'response'] })
-  @IsNotBlank({ message: ValidationErrors.POSTCODE_REQUIRED, groups: ['claimant', 'defendant', 'response'] })
+  @IsDefined({ message: ValidationErrors.POSTCODE_REQUIRED, groups: ['response'] })
+  @IsNotBlank({ message: ValidationErrors.POSTCODE_REQUIRED, groups: ['response'] })
+  @CheckCountry(Country.all(), { groups: ['claimant'] })
+  @CheckCountry(Country.defendantCountries(), { groups: ['defendant'] })
```

Comment:
**Business question:** Should we check defendant address when defendant confirms their details during the response time? What if claimant provided defendant address within allowed range however when defendant tries to respond it happens that his address is outside the allowed range?

Reply:
I asked art this and he said it only matters when issuing the claim. If the defendant updates their address during the response we don't need to check it